python/cugraph/centrality/betweenness_centrality.py

```diff
@@ -26,29 +30,44 @@ def betweenness_centrality(G, k=None, normalized=True,
     G : cuGraph.Graph
         cuGraph graph descriptor with connectivity information. The graph can
         be either directed (DiGraph) or undirected (Graph)
```

Comment:
We should mention that weights stored in the graph are ignored

Reply:
Updated in 50b27a1, also added information about the fact that it is not supported